### DISCUSSION TITLE
Error №20100 on pokemon diamond

### DISCUSSION THREAD

**Discussion by CalibriLight** [mm201/pkmnFoundations#100] [Q&A]
Im using melonds emulator, my wifi speed is faster than light, when i testing connection, everything is nice, but when i am trying connect to GTS, get my friend code, etc. it gives me error 20100 after 10-minutes loading.
Game: pokèmon diamond
Dns: i tried wiimmfi, pkmn-classic's, nothing works. 
P.s. idk where put this issue, let's put here

**Comment by mm201** [comment]
That sounds like your internet. I don't really use melonDS often but I think they have separate NAT and bridged networking modes. Try switching modes. Also try installing winpcap which I think their bridged mode uses.

**Reply by CalibriLight** [reply]
Uhhh, but when my friend tried to connect with pkmnclassic using melonds, everything worked
